IFC4x3/Properties/l/LineID_1LQ0XPbITCVe8zQJfs9vuz/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="LineID_1LQ0XPbITCVe8zQJfs9vuz" Name="LineID" UniqueId="55680859-9527-4c7e-823d-693a76279e3d" PrimaryDataType="IfcIdentifier" />
```

Comment:
I missed this, yes, we have used tag on over situations but this is an IfcElement attribute. Proposal:

- Place ID in Name attribute and use LongName for the descriptive informal name we can encode this as a usage at some level in track Ex or at a more general level.
- This can apply to TrackID also.

Reply:
I just consider this similar as LandID or BuildingID. I think we can shortly discuss this.